kfs-core/src/main/resources/edu/arizona/kfs/fp/spring-fp.xml

```diff
@@ -36,6 +36,17 @@
 
     <bean id="disbursementPayeeLookupableHelperService" scope="prototype" parent="disbursementPayeeLookupableHelperService-parentBean" class="edu.arizona.kfs.fp.businessobject.lookup.UaDisbursementPayeeLookupableHelperServiceImpl" />
 
+	<bean id="paymentMethodGeneralLedgerPendingEntryService" parent="paymentMethodGeneralLedgerPendingEntryService-parentBean"/>
+	<bean id="paymentMethodGeneralLedgerPendingEntryService-parentBean" class="edu.arizona.kfs.fp.service.impl.PaymentMethodGeneralLedgerPendingEntryServiceImpl" abstract="true">
+		<property name="generalLedgerPendingEntryService" ref="generalLedgerPendingEntryService"/>
+		<property name="objectCodeService" ref="objectCodeService"/>
+		<property name="parameterService" ref="parameterService"/>
```

Comment:
I realized that parameter service is not actually being used in this class, so let's remove the property.